src/pages/introduction/essentials/tech-stack/index.md

```diff
@@ -0,0 +1,47 @@
+- [JavaScript versions](https://www.w3schools.com/js/js_versions.asp) and [ECMA2015 (or ES6)](https://www.w3schools.com/js/js_es6.asp)
+- [HTML Basics](https://www.w3schools.com/html/html_intro.asp)
+- [CSS Basics](https://www.w3schools.com/css/css_intro.asp), [syntax](https://www.w3schools.com/css/css_syntax.asp), and [selectors](https://www.w3schools.com/css/css_selectors.asp)
+
+Apart from the basics, the following additional topics will also come in handy. You need not go through each of them right away, but do bookmark them for later as it will help you understand the code snippets much more easily. 
+
+- [Document Object Model or HTML DOM](https://eloquentjavascript.net/14_dom.html) and the global [window](https://www.w3schools.com/jsref/obj_window.asp) and [document](https://www.w3schools.com/jsref/prop_win_document.asp) object
+- [DOM Events](https://javascript.info/introduction-browser-events)
+- [Adding CSS](https://www.w3schools.com/css/css_howto.asp)
+- [CSS layout using flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/)
+- [Promise](https://javascript.info/promise-basics)
+- [Async/await](https://javascript.info/async-await)
+- Different ways of declaring JS functions - [traditional style](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/function) vs using the [fat arrow](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
+
+## Chrome debug tool (CDT)
+One of the easiest ways to debug these technologies is by using the Chrome Debug Tool. We will be using the same within our developer tool called UDT, which stands for the UXP Developer Tool. A high-level understanding of CDT will be very useful for you later.
+
+Please familiarize yourself with the [Chrome Debug tool](https://developer.chrome.com/docs/devtools/overview/), especially the ways to [access DOM](https://developer.chrome.com/docs/devtools/dom/), [modify CSS](https://developer.chrome.com/docs/devtools/css/) and [debug JS](https://developer.chrome.com/docs/devtools/javascript/) by adding breakpoints.
+
+## Frameworks
+So far we have spoken about plain JS, HTML, and CSS. However, the industry offers various frameworks that act as an abstract layer on top of these web technologies. These frameworks aim to provide you with a quicker, and more efficient way of writing reusable code.
+ 
+React, Vue, and Svelte are some of the popular frameworks, that developers often find useful. To use such frameworks is optional and is your personal choice. But, since these are one layer above the plain web technologies, you would require additional tools to run such projects.
+
+- [Node.js](https://nodejs.org/en/): a JavaScript runtime environment. It's often used as a backend server in web environments. Go to the [Node.js download page](https://nodejs.org/en/download/), download the installer for your platform, and run it. This will also install `npm`
```

Comment:
@kareemmyk  What is the version of node/npm that we support? For example. for React we mentioned that it should be v17.0 or below. Is that what is recommended?

Reply:
So any version of Node will work for installing SWC?

Reply:
Whats the maximum version of node we support?